sql/mob_groups.sql

```diff
@@ -10889,7 +10889,7 @@ INSERT INTO `mob_groups` VALUES (10988,4653,82,10000,0,0,15000,9999,83,85,0);
 INSERT INTO `mob_groups` VALUES (10989,4654,222,240,0,0,100000,9999,99,99,0);
 INSERT INTO `mob_groups` VALUES (11001,4657,81,330,128,2797,8000,0,72,76,0);
 INSERT INTO `mob_groups` VALUES (11002,4658,81,330,128,560,4444,0,50,51,0);
-INSERT INTO `mob_groups` VALUES (11003,4655,81,330,128,0,4800,0,60,60,0);
+INSERT INTO `mob_groups` VALUES (11003,4655,81,330,128,4595,4800,0,60,60,0);
```

Comment:
`spawntype` column should be set to 32 (SPAWNTYPE_LOTTERY) rather than 128 (SPAWNTYPE_SCRIPTED)